components/admin/AdminTable.tsx

```diff
@@ -0,0 +1,197 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Heading,
+  Input,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+  InputLeftElement,
+  InputGroup,
+} from '@chakra-ui/react'
+import {
+  EditIcon,
+  DeleteIcon,
+  AddIcon,
+  SearchIcon,
+  ChevronLeftIcon,
+  ChevronRightIcon,
+} from '@chakra-ui/icons'
+import Footer from '../Footer'
+import { useState } from 'react'
+import get from 'lodash/get'
```

Comment:
Why do we need lodash's `get` now? The library is pretty much dead o.O
also note to self: need to check on tree-shaking for all the other deps

Edit: I saw what `get` is for, we can also use optional chaining `?` from ES6